docs/bundles.md

```diff
@@ -0,0 +1,24 @@
+## Bundle Anatomy
```

Comment:
Today, all of the bundle-related docs exist in the README. Would you mind migrating this ` ## Schema Validation` section to that location and adding an entry to the table of contents?

More context: our tech writer is working on migrating all of the UDS CLI docs to a proper website as we speak, so this layout will likely look very different in the coming weeks